exercises/affine-cipher/AffineCipher.fsproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
This file looks slightly different than the other project files. For example, it targets .NET Core 2.2 instead of 2.1. Was it generated automatically using the new exercise script?

For reference what another project file looks like: https://github.com/exercism/fsharp/blob/master/exercises/accumulate/Accumulate.fsproj

Reply:
I used the script, maybe it depends on the sdk installed on my pc? I will edit the fsproj to be like all the others.

Reply:
Great! It must indeed be due to the SDK installed.